src/main/resources/META-INF/plugin.xml

```diff
@@ -21,6 +21,7 @@
     <depends optional="true" config-file="idea-only.xml">com.intellij.modules.java</depends>
     <depends optional="true" config-file="clion-only.xml">com.intellij.modules.clion</depends>
     <depends optional="true" config-file="toml-only.xml">org.toml.lang</depends>
+    <depends optional="true" config-file="plugin-copyright.xml">com.intellij.copyright</depends>
```

Comment:
Let's use the same style for all optional dependency files and name it "copyright-only.xml"

Reply:
Done.